src/test/java/edu/kit/kastel/sdq/artemis4j/UtilitiesTest.java

```diff
@@ -1,113 +0,0 @@
-/* Licensed under EPL-2.0 2023. */
-package edu.kit.kastel.sdq.artemis4j;
-
-import edu.kit.kastel.sdq.artemis4j.api.ArtemisClientException;
-import edu.kit.kastel.sdq.artemis4j.api.artemis.assessment.AssessmentResult;
-import edu.kit.kastel.sdq.artemis4j.api.artemis.assessment.Feedback;
-import edu.kit.kastel.sdq.artemis4j.api.artemis.assessment.Submission;
-import edu.kit.kastel.sdq.artemis4j.client.RestClientManager;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Test;
-
-import java.util.ArrayList;
-import java.util.List;
-
-/**
- * This class does not contain tests as usual. It is used to perform recurring
- * tasks like toggling all exams to submitted.
- */
-@Disabled
-class UtilitiesTest {
```

Comment:
I need all the Utilities to manage different tasks :)

Reply:
Please dont forget to restore the files :)

Reply:
Yes, will do it on Monday